modules/openapi-generator/src/main/resources/JavaJaxRS/api.mustache

```diff
@@ -76,7 +76,7 @@ public class {{classname}}  {
         {{/hasMore}}{{/responses}} })
     public Response {{nickname}}({{#allParams}}{{>queryParams}}{{>pathParams}}{{>headerParams}}{{>bodyParams}}{{>formParams}},{{/allParams}}@Context SecurityContext securityContext)
     throws NotFoundException {
-        return delegate.{{nickname}}({{#allParams}}{{#isFile}}{{paramName}}InputStream, {{paramName}}Detail{{/isFile}}{{^isFile}}{{paramName}}{{/isFile}},{{/allParams}}securityContext);
+        return delegate.{{nickname}}({{#allParams}}{{paramName}}, {{/allParams}}securityContext);
```

Comment:
actually the real problem is the missing `isFormParam `in this condition. the formParams.mustache who generate an InputStream and an FormDataContentDisposition variables are conditionned with the `isFormParam` so this line should be fixed with :  
```
return delegate.{{nickname}}({{#allParams}}{{#isFormParam}}{{#isFile}}{{paramName}}InputStream, {{paramName}}Detail{{/isFile}}{{/isFormParam}}{{^isFile}}{{paramName}}{{/isFile}},{{/allParams}}securityContext);
```
to solve the issue

Reply:
I've changed the template with something inspired by your solution.

I've tried it with the issue's sample and this one :

`
  /Plugin:
    post:
      summary: uploads an image
      description: ''
      operationId: uploadFile
      requestBody:
        content:
          multipart/form-data:
            schema:
              type: object
              properties:
                additionalMetadata:
                  description: Additional data to pass to server
                  type: string
                file:
                  description: file to upload
                  type: string
                  format: binary
`

Reply:
thanks . can you please push your new commit to your branch "2474" to update the PR ?

Reply:
It can be useful. Done.